pipeline/data/L1/code_examples/get-latest-release-name.R

```diff
@@ -0,0 +1,11 @@
+# Programmatically get the list of data releases
+# BBL May 2025
+
+tags <- system2("git",
+                args = c("ls-remote", "--tags", "https://github.com/COMPASS-DOE/data-workflows.git"),
+                stdout = TRUE)
+releases <- gsub("^.*/tags/", "", tags)
```

Comment:
Does this only pull latest release from Github? You think there's a way to also do this for the GDrive?

Reply:
Try running this script in R and you can look at what gets downloaded into `tags` and `releases`

Reply:
```r
> tags <- system2("git",
                args = c("ls-remote", "--tags", "https://github.com/COMPASS-DOE/data-workflows.git"),
                stdout = TRUE)
> tags
[1] "e2be2ef5b0be76adefadafbb8eaf5edca8f60658\trefs/tags/v0-9"
[2] "098b25ee704cf33f003bbbc19720ecc9f2bb8285\trefs/tags/v1-0"
[3] "8adc3ec96f5b5e25ebe8916a876ff0017fb497c3\trefs/tags/v1-1"
[4] "5bff0043a457c78906d5e3d587e12431c7201291\trefs/tags/v1-2"

> releases <- gsub("^.*/tags/", "", tags)
> releases
[1] "v0-9" "v1-0" "v1-1" "v1-2"
```